Dockerfile.mysql

```diff
@@ -0,0 +1,12 @@
+FROM mysql:latest
+
+ENV MYSQL_ROOT_PASSWORD=1234
```

Comment:
Using a hardcoded password for the MySQL root user is a security risk. Consider using environment variables or Docker secrets to manage sensitive information securely.